src/main/java/interface_adapter/delete_clothing_item/DeletePresenter.java

```diff
@@ -0,0 +1,32 @@
+package interface_adapter.delete_clothing_item;
+
+import interface_adapter.ViewManagerModel;
+import use_case.delete_clothing_item.DeleteOutputBoundary;
+import use_case.delete_clothing_item.DeleteOutputData;
+
+public class DeletePresenter implements DeleteOutputBoundary {
+    private final DeleteViewModel deleteViewModel;
+    private ViewManagerModel viewManagerModel;
+
+    public DeletePresenter(DeleteViewModel deleteViewModel, ViewManagerModel viewManagerModel) {
+        this.deleteViewModel = deleteViewModel;
+        this.viewManagerModel = viewManagerModel;
+    }
+
+    @Override
+    public void prepareSuccessView(DeleteOutputData response) {
+        DeleteState deleteState = deleteViewModel.getState();
+        this.deleteViewModel.setState(deleteState);
+        deleteViewModel.firePropertyChanged();
+
+        viewManagerModel.setActiveView(deleteViewModel.getViewName());
```

Comment:
does this mean that when the user deletes a clothing item, they're redirected back to the delete view?

Reply:
My bad. I will change this part.